### PR TITLE
Opportunity Details Page 

### PR DESCRIPTION
**Pull request by Meghana-12**
You can view the page at '/opportunityDetails'
You can view the side Information card at '/interface/opportunitySideCard'
opportunity

### CODE REVIEW

**Comment by Paarmita** [comment]
@Meghana-12 I guess you forgot to add the side information section?

Screenshot 2020-05-16 at 10 19 56 AM

**Comment by Meghana-12** [comment]
> @Meghana-12 I guess you forgot to add the side information section?
> 
> Screenshot 2020-05-16 at 10 19 56 AM

I've just seen the issue update, I'll add the component.

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-OpportunitySideCard/Components/index.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import ShareIcon from '../../../ignitus-Assets/ignitus-Images/img-Png/share.png';
+
+export const InfoCard = ({ location, commitment }: Cardtypes) => (
+  <S.Container>
+    <Card title="Location" description={location} />
+    <Card title="Commitment" description={commitment} />
+    <S.ButtonsWrapper>
+      <S.Button size="large" category="primary">
+        {' '}
+        Apply{' '}
```
Comment (Paarmita) [inline src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-OpportunitySideCard/Components/index.tsx:14]: Can you please add the dynamic feature as well. On clicking the apply button, a message will be displayed saying `Applied successfully` and the button will change to a button with a checkmark icon and text as Applied. YOu can see the screens in Figma for more reference.

File: src/ignitus-OpportunityDetailsPage/Components/index.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    <S.Column>
+      <OpportunityDetailsCard
+        title="PhD Project"
+        subtitle="By The University of Melbourne"
+        description="Church and civil courts in late medieval and early modern Scotland"
+        fund=""
+        src={UniversityOfMelbourne}
+      />
+      <S.Container>
+        <Heading1>Project Description</Heading1>
```
[the remark this answers is not in the document]
Comment (Meghana-12) [reply src/ignitus-OpportunityDetailsPage/Components/index.tsx:19]: okay

**Comment by Meghana-12** [comment]
> Also, remove the university of Melbourne image and use the link https://storage.googleapis.com/ignitus_assets/ig-opportunities/universityOfMelbourne.png.

sure.

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-OpportunitySideCard/Components/index.tsx (hunk 1 above)
Comment (Meghana-12) [reply src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-OpportunitySideCard/Components/index.tsx:14]: Okay!

Comment (Paarmita) [reply src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-OpportunitySideCard/Components/index.tsx:14]: I checked the PR, I guess you missed this.

**Comment by Paarmita** [comment]
@Meghana-12 Can you please check the margin and padding in the design and accordingly change the UI, as I see the heading and paragraph are not left-aligned and other issues.

Also, For the Main heading component, use the `opportunitySaved` component nad remove the funded from it. Instead add the section with location details like in the image. 

Screenshot 2020-05-21 at 8 13 02 PM

File: src/ignitus-OpportunityDetailsPage/Components/index.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            </S.Content>
+          ))}
+          <Paragraph>
+            Candidates should apply for a PhD place in the normal way, stating:
+          </Paragraph>
+          {DetailsRequired.map(content => (
+            <Paragraph key={content}>&#9675;{content}</Paragraph>
+          ))}
+        </S.Content>
+        <Heading1>Funding Notes</Heading1>
```
Comment (Paarmita) [inline src/ignitus-OpportunityDetailsPage/Components/index.tsx:41]: Change it to Heading 2 or 3, heading 1 looks too big here.

Comment (divyanshu-rawat) [reply src/ignitus-OpportunityDetailsPage/Components/index.tsx:41]: Done!

**Comment by divyanshu-rawat** [comment]
@Meghana-12 I have added some new commits feel ree to have a look 👍 :)

**Pull request merged**